velox/dwio/parquet/tests/reader/ParquetTableScanTest.cpp

```diff
@@ -37,13 +37,12 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
 
   void SetUp() {
     registerParquetReaderFactory();
+    HiveConnectorTestBase::SetUp();
+  }
 
-    auto hiveConnector =
-        connector::getConnectorFactory(
-            connector::hive::HiveConnectorFactory::kHiveConnectorName)
-            ->newConnector(
-                kHiveConnectorId, std::make_shared<core::MemConfig>());
-    connector::registerConnector(hiveConnector);
+  void TearDown() {
+    unregisterParquetReaderFactory();
```

Comment:
The `unregisterParquetReaderFactory` also  occurs in the `HiveConnectorTestBase::TearDown` so we don't need it here?

Reply:
fixed.